backend/app/src/main/java/com/moh/phlat/backend/esb/json/OFRelationshipResponse.java

```diff
@@ -0,0 +1,139 @@
+package com.moh.phlat.backend.esb.json;
+
+import java.text.DateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TimeZone;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.StringUtils;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.moh.phlat.backend.esb.boundary.PlrToken;
+import com.moh.phlat.backend.model.Control;
+import com.moh.phlat.backend.model.ProcessData;
+
+public class OFRelationshipResponse implements PlrResponse {
```

Comment:
Similar comments will be applicable to this class as MaintainFacilityResponse

Reply:
Haven't touched OFRelationship or HDS pieces yet; I wanted to focus on Facility for this sprint. These classes are on my plate once Facility is looking good and I can confidently carry over its improvements into these classes too.